pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -115,8 +127,12 @@ type Registry struct {
 	Blocked bool `toml:"blocked,omitempty"`
 	// If true, mirrors will only be used for digest pulls. Pulling images by
 	// tag can potentially yield different images, depending on which endpoint
-	// we pull from.  Forcing digest-pulls for mirrors avoids that issue.
+	// we pull from.  Restricting mirrors to pulls for mirrors avoids that issue.
 	MirrorByDigestOnly bool `toml:"mirror-by-digest-only,omitempty"`
+	// If true, mirrors will only be used for tag pulls. Pulling images by
+	// tag can potentially yield different images, depending on which endpoint
+	// we pull from.  Restricting mirrors to pulls for mirrors avoids that issue.
```

Comment:
I don’t understand what “Restricting mirrors to pulls for mirrors” means — and this should, if anything, explain why one would use `MirrorByTagOnly`, not `MirrorByDigestOnly` (see elsewhere).

But…